File: python/meteroid/models/credit_note_custom_properties_request.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel


@dataclasses.dataclass
class CreditNoteCustomPropertiesRequest(BaseModel):
    """Merge update of a credit note's custom property values (send a key with `null` to remove it).
    Allowed at any status — custom properties stay editable after the credit note is finalized."""

    custom_properties: t.Any

File: go/credit_note_custom_properties_request.go
// this file is @generated
package meteroid

import "encoding/json"

// Merge update of a credit note's custom property values (send a key with `null` to remove it).
// Allowed at any status — custom properties stay editable after the credit note is finalized.
type CreditNoteCustomPropertiesRequest struct {
	CustomProperties json.RawMessage `json:"custom_properties"`
}
